compiler+runtime/src/cpp/jank/runtime/core/seq.cpp

```diff
@@ -934,14 +934,19 @@ namespace jank::runtime
       [=](auto const typed_args) -> native_persistent_string {
         util::string_builder buff;
         buff.reserve(16);
-        runtime::to_string(o, buff);
+        if(!is_nil(o))
+        {
+          runtime::to_string(o, buff);
+        }
         if(0 < sequence_length(typed_args))
```

Comment:
Question for @jeaye: can we remove this conditional? `persistent_list::fresh_seq()` will return `nullptr` and short-circuit the `for`, then we just need to check the count once.

Reply:
Yeah, that makes sense. The for loop with `fresh_seq` should handle this just fine.